### DISCUSSION TITLE
Welcome to WebGames Discussions!

### DISCUSSION THREAD

**Discussion by sarmadhamdani02** [sarmadhamdani02/WebGames#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avinashji1302** [comment]
How can I join the community??

**Reply by sarmadhamdani02** [reply]
Hi @avinashji1302 👋

Well, we do not have any kind of social media platform for our community right now as the community is so small and we have only 14 contributors right now 😅 I really hope we will grow soon.

For now, this GitHub discussion form is where we can discuss stuff and communicate and if you have already asked questions here or contributed to solving some issues, then you are already part of the community. 😊

**Reply by mileswolfallen2** [reply]
can i help?
